wix-embedded-mysql/src/main/java/com/wix/mysql/distribution/fileset/Nix8FileSetEmitter.java

```diff
@@ -18,7 +20,12 @@ public boolean matches(Platform platform, Version version) {
 
     @Override
     public FileSet emit() {
-        return common()
+        return FileSet.builder()
+                .addEntry(Executable, "bin/mysqld")
+                .addEntry(Library, "bin/mysql")
+                .addEntry(Library, "bin/mysqladmin")
+                .addEntry(Library, "bin/my_print_defaults")
+                .addEntry(Library, "share/english/errmsg.sys")
                 .addEntry(FileType.Library, "lib/libssl.so.1.0.0")
```

Comment:
Mysql 8 expects these to be in the /bin dir. Won't work/start on systems that don't have the libs in the bin dir